InnerEye/ML/run_ml.py

```diff
@@ -223,6 +224,16 @@ def setup(self, use_mount_or_download_dataset: bool = True) -> None:
                                                     run_context=RUN_CONTEXT)
         self.checkpoint_handler.download_recovery_checkpoints_or_weights(only_return_path=not is_global_rank_zero())
 
+        if self.azure_config.pretraining_run_recovery_id is not None:
+            run_to_recover = self.azure_config.fetch_run(self.azure_config.pretraining_run_recovery_id.strip())
+            run_recovery_object = RunRecovery.download_all_checkpoints_from_run(self.container,
+                                                                                run_to_recover,
+                                                                                EXTRA_RUN_SUBFOLDER,
+                                                                                only_return_path=not is_global_rank_zero())
+            self.container.extra_downloaded_run_id = run_recovery_object
+        else:
+            self.container.extra_downloaded_run_id = None
```

Comment:
The field name reads a bit strange - it seems to indicate that this is a run ID, but it's a RunRecovery object (which in turn is nothing but a list of Paths).

Reply:
Renamed, see my comment below.